src/App.js

```diff
@@ -1,21 +1,28 @@
-import React from 'react';
-
+import EntryList from "./EntryList.js";
+import data from "./data.json";
 import './App.css';
-function MyList(props) {
-  const items = props.items;
-  const listItems = numbers.map((item) =>
-    <li>{item}</li>
-  );
-  function App() {
-    return (
-      <ul>{listItems}</ul>
-    );
-  }
-  const items = [item1, item2, item3, item4, item5];
-  ReactDOM.render(
-    <MyList items={items} />,
-    document.getElementById('root')
+
+function App() {
+  return (
+    <div className="App">
+      <EntryList fruit="Apple" />
+      <EntryList fruit="Banana" />
+      <EntryList fruit="Orange" />
+    </div>
   );
+
 }
+const items = data.map(function (item) {
```

Comment:
This should be removed

Reply:
As you can see I added data for adding our items 
If I removed the data file where we can add or items names?!

Reply:
You can define array of fruits on the App.js 

const fruits = ["Apples", "Oranges"] 

Pass this array as props to `EntryList` 

Make sure that EntryList render each item in the array using `li` tag